ui-manchette-with-spacetimechart/src/hooks/usePaths.ts

```diff
@@ -16,7 +18,8 @@ const usePaths = (projectPathTrainResult: ProjectPathTrainResult[], selectedTrai
         path.spaceTimeCurves.map((spaceTimeCurve, ind) => ({
```

Comment:
To remove the cast, add the map return type here : 
```suggestion
        path.spaceTimeCurves.map<{
          id: string;
          label: string;
          color: string;
          level: PathLevel;
          points: { time: number; position: number }[];
        }>((spaceTimeCurve, ind) => ({
```

Reply:
Done !